enarx-keep/src/backend/kvm/vm/cpu.rs

```diff
@@ -105,7 +106,16 @@ impl Thread for Cpu {
     fn enter(&mut self) -> Result<Command> {
         match self.fd.run()? {
             VcpuExit::IoOut(port, _) => match port {
-                SYSCALL_TRIGGER_PORT => unimplemented!(),
+                SYSCALL_TRIGGER_PORT => {
+                    let block = self
+                        .keep
+                        .read()
+                        .unwrap()
+                        .address_space
+                        .prefix_mut()
+                        .shared_pages as *mut _ as *mut Block;
```

Comment:
@connorkuehl I didn't figure out how to get the block address otherwise... Maybe it should be precalculated somehow and stored in the Cpu struct.

Reply:
There will be separate shared pages per CPU anyway.

Reply:
Would something like this work: https://github.com/connorkuehl/enarx/commit/0e6eb3282231e4621bb57afe3b5ff6eff79dee7e

Reply:
Yes, thank you! Corrected!